### PR TITLE
Add gets_many to HashClient.

### PR DESCRIPTION
**Pull request by njason**

### CODE REVIEW

**Comment by njason** [comment]
@nichochar I raised the issue that HashClient is missing gets_many here https://github.com/pinterest/pymemcache/issues/136. Could you please review my PR for adding this feature? Thanks.

File: pymemcache/client/hash.py [hunk 1]
```diff
@@ -301,6 +307,11 @@ def get_many(self, keys, *args, **kwargs):
     def gets(self, key, *args, **kwargs):
         return self._run_cmd('gets', key, None, *args, **kwargs)
 
+    def gets_many(self, keys, *args, **kwargs):
+        return self.get_many(keys, True, *args, **kwargs)
```
Comment (nichochar) [inline pymemcache/client/hash.py]: can we write the kwarg explicitely here please? `gets=True`

**Comment by nichochar** [comment]
This looks good, a part from a quick style change. The travis failure I believe is unrelated, due to some issue with gevent 1.1 and python 2.6... Travis always gives us a hard time. I will look into this and try and fix it so you can rebase and we can go ahead and merge your change.

**Comment by njason** [comment]
@nichochar Thanks for reviewing this PR. I have made the `gets=True` style change. My fork is rebased and squashed.

**Comment by njason** [comment]
@nichochar Thank you for merging my PR. Could you kindly update the PyPI package? Let me know if you want me to do the release steps up to uploading the packages step from https://github.com/pinterest/pymemcache/wiki/Release-Checklist

**Comment by nichochar** [comment]
Yep, the pypi package is updated now, `1.4.1`

**Pull request merged**